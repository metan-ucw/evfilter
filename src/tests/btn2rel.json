{
  "filter": "btn2rel",
  "key_up": "Key_Up",
  "key_down": "Key_Down",
  "key_left": "Key_Left",
  "key_right": "Key_Right"
}
