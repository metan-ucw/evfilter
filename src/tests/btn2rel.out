ev->type:  EV_REL
ev->code:  REL_Y (1)
ev->value: 2

ev->type:  EV_SYN

ev->type:  EV_REL
ev->code:  REL_Y (1)
ev->value: -2

ev->type:  EV_SYN

ev->type:  EV_KEY
ev->code:  Key_A (30)
ev->value: KEY_DOWN   (1)

ev->type:  EV_SYN

